docs/pywbemclicmdshelp.rst

```diff
@@ -686,23 +686,26 @@ The following defines the help output for the `pywbemcli connection add --help`
 
 ::
 
-    Usage: pywbemcli connection add [COMMAND-OPTIONS]
+    Usage: pywbemcli connection add [COMMAND-OPTIONS] NAME
 
       Add a new WBEM connection definition from specified options.
 
       Create a new WBEM connection definition in the connections file from the
       specified options. A connection definition with that name must not yet
       exist.
 
-      The current connection remains unchanged by this command.
+      The NAME argument MUST exist. It define the server uri and the unique name
+      under which this server connection information will be stored. A --server
+      or --mock-server must exist because they define the server. All other
+      properties are optional.
 
-      Examples:
+      Adding a connection does not set the new connection as the current
+      connection. Use `connection select` to set a particular stored connection
+      definition as the current connection.
 
         pywbemcli --name newsrv connection add --server https://srv1
 
     Options:
-      -n, --name NAME                 Name for the new WBEM connection definition.
-                                      [required]
```

Comment:
Why is --name removed? It can very well be used to use a previously defined connection defin ition as input for the new connection definition.